cmd/rosa/main.go

```diff
@@ -105,3 +112,30 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func versionCheck() {
```

Comment:
@AlexVulaj to @cben point, I don't think we want this on `rosa version` or `rosa verify rosa`? It's gonna print double the warnings for commands that already include a version check in their output.

Reply:
I've updated our code to ignore version checks on specific commands.